api/src/test/java/ca/bc/gov/educ/api/penrequest/model/PenReqDocumentJpaTests.java

```diff
@@ -1,49 +1,54 @@
 package ca.bc.gov.educ.api.penrequest.model;
 
+import ca.bc.gov.educ.api.penrequest.PenRequestApiResourceApplication;
 import ca.bc.gov.educ.api.penrequest.model.v1.DocumentEntity;
 import ca.bc.gov.educ.api.penrequest.model.v1.PenRequestEntity;
 import ca.bc.gov.educ.api.penrequest.repository.DocumentRepository;
+import ca.bc.gov.educ.api.penrequest.repository.PenRequestRepository;
 import ca.bc.gov.educ.api.penrequest.support.DocumentBuilder;
 import ca.bc.gov.educ.api.penrequest.support.PenRequestBuilder;
+import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringRunner;
 
 import java.util.Optional;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(SpringRunner.class)
-@DataJpaTest
```

Comment:
JPATest stopped working after the upgrade. Had to switch over to SpringBootTest which did not give access to the entity manager. So I ended up rewriting the test to save directly to the repository to make it pass.